common/src/main/java/io/pravega/common/util/Retry.java

```diff
@@ -248,10 +249,11 @@ boolean canRetry(final Throwable e) {
         }
 
         private Class<? extends Throwable> getErrorType(final Throwable e) {
-            if (retryType.equals(CompletionException.class) || throwType.equals(CompletionException.class)) {
+            if (retryType.equals(CompletionException.class) || retryType.equals(ExecutionException.class)
```

Comment:
This looks very familiar to what `ExceptionHelpers.getRealCause` does. Could we move this to a method in that class so that the code sits next to each other?